wcivf/settings/base.py

```diff
@@ -332,7 +332,7 @@ def get_ec2_ip():
     "ENABLE_LAYERS_OF_STATE_FEATURE", False
 )
 LAYERS_OF_STATE_URL = (
-    "https://developers.environments.womblelabs.co.uk/api/v1/layers_of_state/"
+    "https://developers.democracyclub.org.uk/api/v1/layers_of_state/"
```

Comment:
Shouldn't this use `DEVS_DC_BASE` ?

Reply:
Good point, pushed a change to remove this setting and just use the base URL in the app code